File: ApiSuvesaPos/Negocio/Modelo/proveedores/proveedores.vb
Namespace Modelo
    Public Class proveedores
        Public Property CodigoProv As Integer
        Public Property Cedula As String
        Public Property Nombre As String
        Public Property Contacto As String
        Public Property Telefono_Cont As String
        Public Property Observaciones As String
        Public Property Telefono1 As String
        Public Property Telefono2 As String
        Public Property Fax1 As String
        Public Property Fax2 As String
        Public Property Email As String
        Public Property Direccion As String
        Public Property MontoCredito As Decimal
        Public Property Plazo As Integer
        Public Property CostoTotal As Boolean
        Public Property ImpIncluido As Boolean
        Public Property PedidosMes As Integer
        Public Property Utilidad_Inventario As Decimal
        Public Property Utilidad_Fija As Boolean
        Public Property CuentaContable As String
        Public Property DescripcionCuentaContable As String
        Public Property Actualizado As Boolean
        Public Property Inhabilitado As Boolean
        Public Property FEC As Boolean
        Public Property TipoProveedor As String
        Public Property Id_Provincia As Integer
        Public Property Id_Canton As Integer
        Public Property Id_Distrito As Integer
        Public Property OtrasSenas As String
        Public Property CodigoActividad As String
        Public Property CorreoElectronico As String
        Public Property Serie As Boolean
        Public Property Bloqueado As Boolean
        Public Property Cuentas_Bancarias_Proveedor As cuentas_bancarias_proveedor
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxp/proveedores/proveedores.cs
namespace Modelo
{
    public class proveedores
    {
        public int CodigoProv { get; set; }
        public string Cedula { get; set; }
        public string Nombre { get; set; }
        public string Contacto { get; set; }
        public string Telefono_Cont { get; set; }
        public string Observaciones { get; set; }
        public string Telefono1 { get; set; }
        public string Telefono2 { get; set; }
        public string Fax1 { get; set; }
        public string Fax2 { get; set; }
        public string Email { get; set; }
        public string Direccion { get; set; }
        public decimal MontoCredito { get; set; }
        public int Plazo { get; set; }
        public bool CostoTotal { get; set; }
        public bool ImpIncluido { get; set; }
        public int PedidosMes { get; set; }
        public decimal Utilidad_Inventario { get; set; }
        public bool Utilidad_Fija { get; set; }
        public string CuentaContable { get; set; }
        public string DescripcionCuentaContable { get; set; }
        public bool Actualizado { get; set; }
        public bool Inhabilitado { get; set; }
        public bool FEC { get; set; }
        public string TipoProveedor { get; set; }
        public int Id_Provincia { get; set; }
        public int Id_Canton { get; set; }
        public int Id_Distrito { get; set; }
        public string OtrasSenas { get; set; }
        public string CodigoActividad { get; set; }
        public string CorreoElectronico { get; set; }
        public bool Serie { get; set; }
        public bool Bloqueado { get; set; }
        public cuentas_bancarias_proveedor Cuentas_Bancarias_Proveedor { get; set; }
    }
}

